### ISSUE TITLE
feature: commands to check if logged in

### ISSUE THREAD

**Issue by andykais** [bitwarden/cli#27]
Commands  in the fashion of 
```bash
bw login --check
# exit code 0 if logged in or 1 if logged out
```

This will be useful for building commands on top of the cli. Specifically with this package I built [bitwarden-dmenu](https://github.com/andykais/bitwarden-dmenu), it would be useful to know if the user has logged in before trying to unlock so I can send a useful error to the user

**Comment by Mange** [comment]
That would be great. I had to end up guess from the error message string in my script, which is very brittle.

**Comment by blacs30** [comment]
Would like to see it too, would be very helpful

**Comment by ewjoachim** [comment]
Integration the cli from a Python script and windering all the same.

That being said, there has to be some information persisted on the disk when bitwarden is logged in, maybe we could check for that (wating for a proper --check to be implemented)

**Comment by ewjoachim** [comment]
Bingo :) Took me a while to explore the code base but allow me the occasion to say once I had found the jslib repo, it was really nice and readable. Many pointers, but at least, each module seems to have a unique dedicated purpose. Thank you.

https://github.com/bitwarden/cli/blob/783e7fc8348d02853983211fa28dd8448247ba92/src/bw.ts#L67-L75

**Comment by ewjoachim** [comment]
Hahaha the doc is actually giving this information too. A shame I only found it by searching for `BITWARDENCLI_APPDATA_DIR` 😂 

https://help.bitwarden.com/article/where-is-data-stored-computer/#cli

**Comment by andykais** [comment]
in that case, I would still like to request that either a login check command is added, or the ability to ask the cli where the appdata is. A program built on top of `bw` should not have to keep track of four possible locations of internal config data. If its allowed, I am happy to take a stab at this and PR something

**Comment by kspearrin** [comment]
Resolved in #47

**Comment by ewjoachim** [comment]
Yay thank you and @estheruary :clap: :tada: :+1: ! Any idea when that might land in a release ?
